### PR TITLE
[MIG] account_partner_required: Migration to 13.0

### PR DESCRIPTION
**Pull request by lembregtse**
Has an OCA dependency on account_menu which is currently in PR: 

https://github.com/OCA/account-financial-tools/pull/889

### CODE REVIEW

**Comment by kittiu** [comment]
Functional tested. LGTM

**Review by kittiu — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]
LGTM 👍 @lembregtse could you solve conflicts, please?

File: account_partner_required/tests/test_account_partner_required.py [hunk 1]
```diff
@@ -18,6 +18,11 @@ def setUp(self):
         self.account_type_obj = self.env['account.account.type']
         self.move_obj = self.env['account.move']
         self.move_line_obj = self.env['account.move.line']
+        self.general_journal = self.env['account.journal'].create({
```
Comment (alexis-via) [inline account_partner_required/tests/test_account_partner_required.py]: Why do you create this journal... seems you don't use it!

**Pull request closed without merging**